README.md

```diff
@@ -62,12 +62,21 @@ Create a new Volto project by using the `@plone/generator-volto` utility.
 
 It will bootstrap a Volto project in a folder of your choice with all the required
 boilerplate to start customizing your Volto site.
-```
-sudo npm install -g yo @plone/generator-volto
+
+```shell
+npm install -g yo @plone/generator-volto
 yo @plone/volto
 ```
-follow the prompts questions, provide `myvoltoproject` as project name then, when it finishes:
+in case npm install commands throws an error, which would be most likely due to a permissions issue in your home directory, you will need to reclaim ownership of the .npm directory.
```

Comment:
Please insert a blank line above this sentence.

Also here is a suggestion to take advantage of inline literals and clever way to represent admonitions using plain old Markdown syntax. (h/t https://stackoverflow.com/a/64254203/2214933)

```suggestion
> 💡 **HINT** In case the `npm install` command throws an error, which would be most likely due to a permissions issue in your home directory, you will need to reclaim ownership of the `.npm` directory.
```

Unfortunately the `README.md` does not support MyST syntax because it is a stand alone Markdown document, living separately from the documentation. If it did support MyST syntax, I would use a [`hint` admonition](https://sphinx-themes.org/sample-sites/sphinx-book-theme/kitchen-sink/admonitions/#hint).